v0/destinations/active_campaign/transform.js

```diff
@@ -67,8 +67,11 @@ const customTagProcessor = async (message, category, destination) => {
     message,
     MAPPING_CONFIG[category.name]
   );
-  contactPayload.firstName = getFieldValueFromMessage(message, "firstName");
-  contactPayload.lastName = getFieldValueFromMessage(message, "lastName");
+  contactPayload.firstName =
+    getFieldValueFromMessage(message, "firstName") || undefined;
+  contactPayload.lastName =
+    getFieldValueFromMessage(message, "lastName") || undefined;
+
```

Comment:
```suggestion
  contactPayload.firstName =
    getFieldValueFromMessage(message, "firstName");
  contactPayload.lastName =
    getFieldValueFromMessage(message, "lastName");
contactPayload = removeUndefinedAndNullValues(contactPayload);
```

removeUndefinedAndNullValues functions removes all the undefined and null properties from an object, this can be imported from `v0/util/index.js`

Reply:
Feedback implemented @utsabc